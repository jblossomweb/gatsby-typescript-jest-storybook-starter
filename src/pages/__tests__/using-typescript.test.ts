import snapshotTests from "../../../utils/story-snapshot-tests"

jest.mock("../../templates/default")
jest.mock("../../components/seo")

import { stories } from "./using-typescript.stories"

describe("pages/using-typescript", () => {
  snapshotTests(stories)
})
