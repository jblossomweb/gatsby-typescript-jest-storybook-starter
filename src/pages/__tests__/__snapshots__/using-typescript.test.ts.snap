// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/using-typescript build time matches snapshot 1`] = `
Array [
  <header
    style={
      Object {
        "background": "rebeccapurple",
        "marginBottom": "1.45rem",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "0 auto",
          "maxWidth": 960,
          "padding": "1.45rem 1.0875rem",
        }
      }
    >
      <h1
        style={
          Object {
            "margin": 0,
          }
        }
      >
        <a
          href="/"
          style={
            Object {
              "color": "white",
              "textDecoration": "none",
            }
          }
        >
          Test Title
        </a>
      </h1>
    </div>
  </header>,
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "0 1.0875rem 1.45rem",
      }
    }
  >
    <main>
      <h1>
        Gatsby supports TypeScript by default!
      </h1>
      <p>
        This means that you can create and write 
        <em>
          .ts/.tsx
        </em>
         files for your pages, components etc. Please note that the 
        <em>
          gatsby-*.js
        </em>
         files (like gatsby-node.js) currently don't support TypeScript yet.
      </p>
      <p>
        For type checking you'll want to install 
        <em>
          typescript
        </em>
         via npm and run 
        <em>
          tsc --init
        </em>
         to create a 
        <em>
          .tsconfig
        </em>
         file.
      </p>
      <p>
        You're currently on the page "
        /using-typescript/
        " which was built on
         
        2020-07-10 02:22 am UTC
        .
      </p>
      <p>
        To learn more, head over to our
         
        <a
          href="https://www.gatsbyjs.org/docs/typescript/"
        >
          documentation about TypeScript
        </a>
        .
      </p>
      <a
        href="/"
      >
        Go back to the homepage
      </a>
    </main>
    <footer>
      © 
      2020
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
        target="_blank"
      >
        Gatsby
      </a>
    </footer>
  </div>,
]
`;

exports[`pages/using-typescript path matches snapshot 1`] = `
Array [
  <header
    style={
      Object {
        "background": "rebeccapurple",
        "marginBottom": "1.45rem",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "0 auto",
          "maxWidth": 960,
          "padding": "1.45rem 1.0875rem",
        }
      }
    >
      <h1
        style={
          Object {
            "margin": 0,
          }
        }
      >
        <a
          href="/"
          style={
            Object {
              "color": "white",
              "textDecoration": "none",
            }
          }
        >
          Test Title
        </a>
      </h1>
    </div>
  </header>,
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "0 1.0875rem 1.45rem",
      }
    }
  >
    <main>
      <h1>
        Gatsby supports TypeScript by default!
      </h1>
      <p>
        This means that you can create and write 
        <em>
          .ts/.tsx
        </em>
         files for your pages, components etc. Please note that the 
        <em>
          gatsby-*.js
        </em>
         files (like gatsby-node.js) currently don't support TypeScript yet.
      </p>
      <p>
        For type checking you'll want to install 
        <em>
          typescript
        </em>
         via npm and run 
        <em>
          tsc --init
        </em>
         to create a 
        <em>
          .tsconfig
        </em>
         file.
      </p>
      <p>
        You're currently on the page "
        /a-different-path
        " which was built on
         
        2020-07-11 03:26 am UTC
        .
      </p>
      <p>
        To learn more, head over to our
         
        <a
          href="https://www.gatsbyjs.org/docs/typescript/"
        >
          documentation about TypeScript
        </a>
        .
      </p>
      <a
        href="/"
      >
        Go back to the homepage
      </a>
    </main>
    <footer>
      © 
      2020
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
        target="_blank"
      >
        Gatsby
      </a>
    </footer>
  </div>,
]
`;

exports[`pages/using-typescript sample matches snapshot 1`] = `
Array [
  <header
    style={
      Object {
        "background": "rebeccapurple",
        "marginBottom": "1.45rem",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "0 auto",
          "maxWidth": 960,
          "padding": "1.45rem 1.0875rem",
        }
      }
    >
      <h1
        style={
          Object {
            "margin": 0,
          }
        }
      >
        <a
          href="/"
          style={
            Object {
              "color": "white",
              "textDecoration": "none",
            }
          }
        >
          Test Title
        </a>
      </h1>
    </div>
  </header>,
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "0 1.0875rem 1.45rem",
      }
    }
  >
    <main>
      <h1>
        Gatsby supports TypeScript by default!
      </h1>
      <p>
        This means that you can create and write 
        <em>
          .ts/.tsx
        </em>
         files for your pages, components etc. Please note that the 
        <em>
          gatsby-*.js
        </em>
         files (like gatsby-node.js) currently don't support TypeScript yet.
      </p>
      <p>
        For type checking you'll want to install 
        <em>
          typescript
        </em>
         via npm and run 
        <em>
          tsc --init
        </em>
         to create a 
        <em>
          .tsconfig
        </em>
         file.
      </p>
      <p>
        You're currently on the page "
        /using-typescript/
        " which was built on
         
        2020-07-11 03:26 am UTC
        .
      </p>
      <p>
        To learn more, head over to our
         
        <a
          href="https://www.gatsbyjs.org/docs/typescript/"
        >
          documentation about TypeScript
        </a>
        .
      </p>
      <a
        href="/"
      >
        Go back to the homepage
      </a>
    </main>
    <footer>
      © 
      2020
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
        target="_blank"
      >
        Gatsby
      </a>
    </footer>
  </div>,
]
`;
